frontend/views/components/GroupSettings.vue

```diff
@@ -37,6 +37,7 @@ import sbp from '~/shared/sbp.js'
 import { toPercent } from '@view-utils/filters.js'
 import { LOAD_MODAL } from '@utils/events.js'
 
+// TODO: there is another file called GroupSettings in pages/, rename this one (and move it?)
```

Comment:
I'm okay with it, do it!

Reply:
This is a useful temporary file until we move all the functionalities present in that component somewhere (only partially done atm).

Reply:
I'll leave this up to you @sandrina-p and @pieer to figure out. :)